src/android/FirebaseAuthenticationPlugin.java

```diff
@@ -246,4 +248,42 @@ private static PluginResult getProfileResult(FirebaseUser user) {
             return new PluginResult(PluginResult.Status.ERROR, e.getMessage());
         }
     }
+
+    @CordovaMethod
+    private void updateProfile(JSONObject params, CallbackContext callbackContext) {
+        FirebaseUser user = this.firebaseAuth.getCurrentUser();
+
+        if (user == null) {
+            callbackContext.error("User is not authorized");
+        }
+        else {
+            UserProfileChangeRequest request = this.toUserProfileChangeRequest(params);
+            if (request == null) {
+                callbackContext.success();
+                return;
+            }
+
+            user.updateProfile(request)
+                    .addOnCompleteListener(cordova.getActivity(), createCompleteListener(callbackContext));
+        }
+    }
+
+    private UserProfileChangeRequest toUserProfileChangeRequest(JSONObject jsonObject) {
+        if (!jsonObject.has("displayName") && !jsonObject.has("photoURL"))
```

Comment:
Sure. Will remove the check.

I am not sure what happens when `UserProfileChangeRequest` doesn't contain any fields. There is no documentation.
It could be error, or just complete with no change.